### DISCUSSION TITLE
Quarkus Runtime Performance

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#25161] [Quarkus Blog/Website]
# Quarkus Runtime Performance

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/runtime-performance/

**Comment by dameng58** [comment]
1、How to reduce the use of reflections in build-time and arc?

**Comment by javawithrocky** [comment]
I looked but couldn't find how 40 concurrent connections was configured? Can you please point me to it?

**Comment by gian1200** [comment]
Is there an updated version of this with Quarkus 3?
